backend/mcp_backend/servers/template_department_sharepoint_server.py

```
@@ -0,0 +1,86 @@
+from generic_sharepoint_server_multi_dept import main as run_generic_server
+
+
+async def main():
+    """Main function to run the {DEPT_NAME} SharePoint MCP server"""
+    try:
+        logging.info("Starting {DEPT_NAME} SharePoint MCP Server")
+        # Initialize the generic server with {DEPT_UPPER} department prefix
+        await run_generic_server(department_prefix="{DEPT_UPPER}")
+
+    except Exception as e:
+        logging.error(f"Error starting {DEPT_NAME} SharePoint server: {e}")
```

Ruff (0.14.7)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)